swarm/src/lib.rs

```diff
@@ -1153,7 +1153,9 @@ impl NetworkBehaviour for DummyBehaviour {
     type OutEvent = void::Void;
 
     fn new_handler(&mut self) -> Self::ProtocolsHandler {
-        protocols_handler::DummyProtocolsHandler::default()
+        protocols_handler::DummyProtocolsHandler {
+            keep_alive: KeepAlive::Yes
```

Comment:
Sorry for being a bit over careful, but instead of silently doing this change, how about:

- Add a `keep_alive` field to `DummyBehaviour`.
- Add a method `new_with_keep_alive` method to `DummyBehaviour` returning a `DummyBehaviour { keep_alive: KeepAlive::Yes }`. The `Default::default` would be `KeepAlive::No`.
- In `DummyBehaviour::new_handler` pass `self.keep_alive` to `DummyProtocolsHandler`.

This might safe someone out there a couple of hours debugging a higher test using this.

Reply:
Sure!